src/components/Form/Form.css

```diff
@@ -1,17 +1,59 @@
 .form {
-  align-items: center;
-  background-color: pink;
+  height: 150px;
+  width: auto;
   display: flex;
   flex-direction: column;
+  align-items: center;
   justify-content: space-around;
-  height: 200px;
-  width: 100%;
+  background-color: rgba(245, 245, 245, 0.90);
+  color: #2c3e50;
+  border-radius: 1rem;
+  margin: 1.5rem 0rem .5rem 0rem;
+  padding: 2rem;
 }
 
 .dropdown {
+  width: 250px;
+  background-color: rgba(245, 245, 245, 0.95);
+  font-size: 18px;
+  text-align-last: center;
+  border-radius: .35rem;
```

Comment:
I think best practice is to always include the leading 0 on decimals, we can also update that on our final review.

Reply:
Made this adjustment 👍